### ISSUE TITLE
Filter IPs when populating variables with katprep_populate

### ISSUE THREAD

**Issue by stdevel** [stdevel/katprep#35]
Currently, the first IP found is used when populating host information. If a virtual machine has multiple network interfaces or protocols (*IPv4/6*), this could be a problem:

```
$ katprep_populate -C giertz.auth -s pinkepank.test.loc --virt-uri vc.test.loc --virt-type pyvmomi --mon-url http://mon.test.loc/test/icinga --mon-type nagios -n -d
...
DEBUG:katprep_populate:HYPERVISOR: Found VM 'pinkepank.giertz.loc' with IP 'fe6a:251e:328:1000::2'
```

It would be great to have IPv4/6 filters or the possibility to match multiple IPs.

**Comment by stdevel** [comment]
Thinks have changed with latest pyVmomi versions, auto-detection is more reliable; but I'm implementing a ipv6-only parameter anyway. Currently developing in `feature-ip_issue35` branch.

**Comment by stdevel** [comment]
Fixed in commit 2f97ad010abbef1cd4abe724f3eadba907866d17.
